### ISSUE TITLE
make with -DENABLE_FILTERAUDIO=0 fails

### ISSUE THREAD

**Issue by flussence** [uTox/uTox#809]
Commit: 53cb07945897
```
 $ cmake -DENABLE_FILTERAUDIO=0 && make
...
[ 97%] Building C object CMakeFiles/utox.dir/src/utox.c.o
[ 98%] Building C object CMakeFiles/utox.dir/src/window.c.o
[100%] Linking C executable utox
/usr/lib/gcc/x86_64-pc-linux-gnu/5.4.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -lfilteraudio
```
The linker flag seems to be hardcoded at CMakeLists.txt line 261.

**Comment by robinlinden** [comment]
This will be (is?) fixed in #801.

**Comment by GrayHatter** [comment]
@flussence are you sure -DENABLE_FILTERAUDIO is what you want? (It will be if you're using the develop branch)

Also, I've been using =OFF instead of =0, maybe that's the issue?

**Comment by GrayHatter** [comment]
is it a static build?

**Comment by robinlinden** [comment]
We ran into this problem in #801. fluessence is right about it being hardcoded and OFF is equivalent to 0 in cmake truthiness. This will be fixed after #801 is merged.
